src/Apps/Order2/Routes/Checkout/__tests__/Order2CheckoutRoute.jest.tsx

```diff
@@ -501,10 +631,23 @@ describe("Order2CheckoutRoute", () => {
           userEvent.click(screen.getByText("Delivery"))
         })
         expect(screen.queryByText("Shipping method")).toBeInTheDocument()
+        expectTrackedEvents({ mockTrackEvent }, [
+          {
+            action: "clickedFulfillmentTab",
+            context_page_owner_id: "order-id",
+            flow: "Buy now",
+            method: "Delivery",
+          },
+        ])
       })
     })
 
     describe("within the payment section", () => {
+      it.todo(
+        // TODO: Example of this assertion is above for clickedChangeShippingAddress
+        "Allows clicking the edit button to change payment method",
+      )
```

Comment:
TIL 🌟

Reply:
yeah this test felt like it was going to be pretty tedious to write, and so much of the payment element is mocked out and there is a very similar test right above. I'm thinking it could be easier to do it when we implement wire transfer or just have a better sense of how to test around that stripe element.